ci/Jenkinsfile.tests

```diff
@@ -58,6 +60,28 @@ pipeline {
       } } }
     }
 
+    stage('postgres tests') {
```

Comment:
The whole Jenkinsfile is for tests, and the job is called `tests`. I don't think yet another `tests` in the stage names. Just call the stage `PostgreSQL`.